.github/workflows/nox.yml

```diff
@@ -77,10 +77,10 @@ jobs:
           python-versions: "${{ matrix.python-versions }}"
       - name: Set up nox environments
         run: |
-          OTHER_ANTSIBULL_MODE=git nox -v -e "${{ matrix.session }}" ${{ matrix.codecov && 'coverage' || '' }} --install-only
+          OTHER_ANTSIBULL_MODE=local nox -v -e "${{ matrix.session }}" ${{ matrix.codecov && 'coverage' || '' }} --install-only
       - name: "Run nox -e ${{ matrix.session }}"
         run: |
-          OTHER_ANTSIBULL_MODE=git nox -v -e "${{ matrix.session }}" --reuse-existing-virtualenvs --no-install
+          OTHER_ANTSIBULL_MODE=local nox -v -e "${{ matrix.session }}" --reuse-existing-virtualenvs --no-install
```

Comment:
This works, of course, but what about adding `OTHER_ANTSIBULL_MODE=local` to `env` at the top instead? I think that might be cleaner.

Reply:
I agree. Implemented in ac91c9ee0413b58c91dbfca5361badcbd2449403.